content/community/contributing-code/Bugs.adoc

```diff
@@ -14,10 +14,10 @@ begin making commits to the project.
 
 * First, determine which base branch your bug fix should use. 
 
-{{%notice info%}}If the bug you're fixing is present in more than one version, then you will need to make a separate pull request for each branch: `hotfix`, `hotfix-7.8.x`, and `hotfix-7.10.x`.{{%/notice%}}
+{{%notice info%}}If the bug you're fixing is present in hotfix-7.8.x then you will need to make two seperate pull requests, one to hotfix-7.8.x and one to hotfix-7.10.x. The hotfix-7.10.x branch will be merged into hotfix and the develop branch so no more duplicates are necessary.`.{{%/notice%}}
```

Comment:
typo `separate`

I also suggest surrounding branch names in backticks or in asterisks (for bold).

Reply:
Updated, will also take a look at turning this into a flow-chart.